src/ucs/config/global_opts.c

```diff
@@ -87,7 +87,7 @@ static ucs_config_field_t ucs_global_opts_table[] = {
  {"LOG_FILE_ROTATE", "0",
   "The maximal number of backup log files that could be created to save logs\n"
   "after the previous ones (if any) are completely filled.",
-  ucs_offsetof(ucs_global_opts_t, log_file_rotate), UCS_CONFIG_TYPE_UINT},
+  ucs_offsetof(ucs_global_opts_t, log_file_rotate), UCS_CONFIG_TYPE_INT},
```

Comment:
i don't think need to change this to INT, it can't be negative, just change internal variables

Reply:
done